build/debian-hyperkube-base/Makefile

```diff
@@ -17,15 +17,15 @@
 # Usage:
 #   [ARCH=amd64] [REGISTRY="staging-k8s.gcr.io"] make (build|push)
 
-REGISTRY?=staging-k8s.gcr.io
+REGISTRY?=gcr.io/k8s-staging-build-image
 IMAGE?=$(REGISTRY)/debian-hyperkube-base
-TAG=0.12.2
+TAG?=v1.0.0
```

Comment:
is there a reason to make this v1.0.0? will that trick people into using it when they shouldn't? ("It's GA!")

Reply:
@liggitt -- mostly because there are barely calculable amount of changes between the images and I want to make it clear that there might not be a SemVer-ish compatibility guarantee between package versions in both.

Reply:
ok... I'm fuzzy on the implications of this in a cherry-pick back to 1.17/1.16... would those also update their own hyperkube base images?

Reply:
This PR is going to do two things:
- update CNI plugins to v0.8.6
- trigger a staging build for debian-hyperkube-base@v1.0.0

Then we'll need to:
- promote the image to prod
- open another 1.18 PR to actually use the new image in the relevant file(s) (specified in `build/dependencies.yaml`)

We should only enable debian-hyperkube-base image building for `release-1.18`, because we don't actually need to build the image on every branch (just the latest branch that still supports hyperkube).

The cherry picks for 1.17 and 1.16 will instead include just:
- update CNI plugins to v0.8.6
- use the new image in the relevant file(s) (specified in `build/dependencies.yaml`)

Reply:
ah, ok. would it make sense to split the hyperkube-base build bit we don't intend to pick back to 1.17/1.16 into a separate PR so this one can include the `actually use the new image in the relevant file(s) (specified in build/dependencies.yaml)` bit and match 1.17/1.16 as closely as possible?

Reply:
@liggitt -- I actually purposely did it this way.

If we do the image build as a separate PR, it means that we'll have to build that one using CNI plugins @ v0.8.5, which means roughly 5 PRs related to 1.18, instead of 2.

Reply:
Concretely, the 1.18 PRs would end up being:
build new debian-hyperkube-base (with CNI plugins v0.8.5) --> promote --> CNI v0.8.6 bump (which builds another new image) --> promote --> PR to use new debian-hyperkube-base image